test/integration/smart_answer_flows/uk_benefits_abroad_test.rb

```diff
@@ -198,26 +198,17 @@ class UKBenefitsAbroadTest < ActiveSupport::TestCase
         end
       end
 
-      context "answer Jersey or Guernsey" do
-        setup do
+      context "answer Jersey" do
+        should "go to Channel Islands outcome" do
```

Comment:
Is this the right assertion? In the deleted code on the left, we have:

```
          should "go to Channel Islands outcome" do	
            add_response "one_year_or_less"	
            assert_current_node :jsa_channel_islands_outcome	
          end
```

Whereas in this new code, we're asserting `jsa_not_entitled_outcome` rather than `jsa_channel_islands_outcome`.